hnn_core/network.py

```diff
@@ -509,8 +557,99 @@ def plot_input(self, ax=None, show=True):
             plt.show()
         return ax.get_figure()
 
-    def plot_spikes(self, ax=None, show=True):
-        """Plot the spiking activity for each cell type.
+
+class Spikes(object):
```

Comment:
can you add it to the public API here: https://45-168215891-gh.circle-artifacts.com/0/html/api.html# ? We are not yet finalized as we haven't made a release but I think it's nice to have it there so we have an overview of the documentation

Reply:
Starting from Python 3, you don't need to inherit from `object`. But I guess we have done this elsewhere, so let's remove it in another pull request